Code/Framework/AzToolsFramework/AzToolsFramework/API/PythonLoader.cpp

```diff
@@ -32,7 +32,49 @@ namespace AzToolsFramework::EmbeddedPython
         UnloadRequiredModules();
     }
 
-    AZ::IO::FixedMaxPath PythonLoader::GetPythonVenvPath(AZ::IO::PathView thirdPartyRoot, AZStd::string_view engineRoot)
+    AZ::IO::FixedMaxPath PythonLoader::GetDefault3rdPartyPath(bool createOnDemand)
+    {
+        AZ::IO::FixedMaxPath thirdPartyEnvPathPath;
+        static constexpr const char* env3rdPartyKey = "LY_3RDPARTY_PATH";
+
+        char env3rdPartyPath[AZ::IO::MaxPathLength] = {'\0'};
+        auto envOutcome = AZ::Utils::GetEnv(AZStd::span(env3rdPartyPath), env3rdPartyKey);
+        if (envOutcome && (strlen(env3rdPartyPath) > 0))
+        {
+            // If so, then use the path that is set as the third party path
+            thirdPartyEnvPathPath = AZ::IO::FixedMaxPath(env3rdPartyPath).LexicallyNormal();
+        }
+        else
+        {
+            auto manifestPath = AZ::Utils::GetO3deManifestDirectory();
+            thirdPartyEnvPathPath = AZ::IO::FixedMaxPath(manifestPath) / "3rdParty";
+        }
+
+        AZStd::string thirdPartyPathString = thirdPartyEnvPathPath.String();
+        if ((!AZ::IO::SystemFile::IsDirectory(thirdPartyPathString.c_str())) && createOnDemand)
```

Comment:
The AZ::IO::Path function has a [.c_str,](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/IO/Path/Path.h#L556) so there is no need to convert the fixed_string into an AZStd::string.
```suggestion

        if ((!AZ::IO::SystemFile::IsDirectory(thirdPartyEnvPathPath.c_str())) && createOnDemand)
```